documentation/book/assembly-kafka-bridge-replicas.adoc

```diff
@@ -0,0 +1,22 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-bridge.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-kafka-bridge-replicas-{context}']
+
+= Replicas
+
+Kafka Bridge can run multiple nodes.
+The number of nodes is defined in the `KafkaBridge` resource.
+Running a Kafka Bridge with multiple nodes can provide better availability and scalability.
+However, when running Kafka Bridge on {ProductPlatformName} it is not absolutely necessary to run multiple nodes of Kafka Bridge for high availability.
```

Comment:
I am not sure about this section. Due to our problem with consumers which have to connect to the same bridge instance where the consumer were created, as we mentioned in the past we cannot run the bridge with multiple replicas but having just one and for HA maybe more bridge deployment with just one replica. @scholzj wdyt?

Reply:
Well, having multiple bridge instances with a single replica will not be HA. Because either you have one service handling them all in case you have the same issues with case you have the same issues as with single instance in multiple replicas. Or you use different services in which case you can hardly call it HA because clients will need to reconfigure to fail over.

That said, maybe this section of the docs should do more about describing the problems with multiple replicas than explain around HA.